src/common/service/NetRemoteService.cxx

```diff
@@ -389,6 +427,76 @@ NetRemoteService::WifiAccessPointDisable([[maybe_unused]] ::grpc::ServerContext*
     return grpc::Status::OK;
 }
 
+using Microsoft::Net::Wifi::Dot11PhyType;
+
+::grpc::Status
+NetRemoteService::WifiAccessPointSetPhyType([[maybe_unused]] ::grpc::ServerContext* context, const ::Microsoft::Net::Remote::Wifi::WifiAccessPointSetPhyTypeRequest* request, ::Microsoft::Net::Remote::Wifi::WifiAccessPointSetPhyTypeResult* response)
+{
+    using Microsoft::Net::Wifi::Ieee80211AccessPointCapabilities;
+
+    LOGD << std::format("Received WifiAccessPointSetPhyType request for access point id {}", request->accesspointid());
+
+    WifiAccessPointOperationStatus status{};
+    status.set_code(WifiAccessPointOperationStatusCode::WifiAccessPointOperationStatusCodeSucceeded);

```

Comment:
We should move this to the end of the function to avoid any possibility of it being set on an error, and just generally delay doing anything until it's needed.